.github/workflows/integration_tests.yml

```diff
@@ -0,0 +1,68 @@
+name: Integration Tests
+
+on:
+  pull_request:
+    paths:
+      - 'docs/example-library/**'
+      - 'integration/**'
+  workflow_dispatch:
+    inputs: {}
+
+jobs:
+  validate:
+    strategy:
+      matrix:
+        include:
+        - { integration_test: "meltano-basics", needs_postgres: false }
+        - { integration_test: "meltano-run", needs_postgres: true }
```

Comment:
Can we add a reminder somewhere in `CONTRIBUTING.md`?

 or perhaps contributor-related docs in `docs/example-library/README.md` that mention registering the example here in the GitHub CI matrix?

Reply:
👍 the readme already links out to the contributor docs - was planning to document it there if this all looked good.